src/splunk_otel/__init__.py

```diff
@@ -11,4 +11,16 @@
 #  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
+from splunk_otel.configurator import SplunkConfigurator
+from splunk_otel.distro import SplunkDistro
 
+
+def init_otel():
```

Comment:
Open to suggestions on this name.

Reply:
`init_splunk_otel`?

Reply:
Yeah that's more specific and probably better. Done.